### ISSUE TITLE
Navbar Issue

### ISSUE THREAD

**Issue by ank1traj** [All-About-Programming-Community/HackTheWinter-Official-Website#7]
Due to responsive nav bar page is breaking with the desktop screen size. On going any page it's showing the white glaze and cross button. 

@Sunny-64 look into this. 

P.S - Before merging & after merging please look into the deployment also. Priority doesn't means not following the basic checklist before and after merging the code. Also, sometimes code following basic software architecture & follows principles of SOLID, DRY etc. can break also. 

![Screenshot from 2022-12-06 15-24-03](https://user-images.githubusercontent.com/32160084/205878867-c9c83f3e-2e9f-49ca-ad92-3807326622c4.png)

**Comment by Sunny-64** [comment]
Sorry for the inconvenience. I will work on this issue right away. I, will make sure to test it properly from next time.

**Comment by Suvraneel** [comment]
@ank1traj  
Well, I did notice that & address that, just not in GitHub.
Attaching screenshots for reference:
![image](https://user-images.githubusercontent.com/63473496/206851281-0b0bf42b-7d58-47a3-8060-80ef6294c783.png)


Nevertheless your inputs are appreciated, just not irrelevant digs based off any grudges you hold.

**Comment by Suvraneel** [comment]
Also, just to clarify ...
This comment was a response 
![image](https://user-images.githubusercontent.com/63473496/206851535-f62242aa-2cae-4a54-8325-6332838a1967.png)

to this:
![image](https://user-images.githubusercontent.com/63473496/206851558-3ad0b586-847f-4250-9e53-ce86f7593b56.png)


He asked who's gonna merge... I replied I will, after I make all necessary changes... In anyway not relating to your review

**Comment by Suvraneel** [comment]
@ank1traj  
About the merge... I don't consider one pop-up as a breaking change esp. since we'll be commenting out the navbar in production for the time being as per @its-puneet 's directions of landing page only as priority.  

That being said...
Please continue to review & be a critic to their work since I'll be unavailable for the month.

Thanks & Cheers

**Comment by ank1traj** [comment]
@Suvraneel It was not reply to you or anyone. It was just the message for everyone that please go with the deployment once too. It might break somewhere if it's passing every software framework. 

Didn't have anything towards you or @Sunny-64. If you're thinking that it was against you by any means then you're too wrong. But, still if you're thinking in that way then I am really sorry.

**Comment by Suvraneel** [comment]
@ank1traj  Kindly explain me the relevance of this here:
![image](https://user-images.githubusercontent.com/63473496/206851759-c4c39591-1e50-4e49-89cf-3154f05cb4ce.png)
  
  
  What has SOLID & DRY principle to do with breaking? (Although I dont consider that a breaking change, but anyway)  
  We're all volunteers here.  I did notice & address that but even if I didn't... `Check production before merging` would have sufficed

**Comment by ank1traj** [comment]
It was PostScript that before & after merging please check the deployment. Each line can have multiple meaning and it's totally on you which one you wanna take.

**Comment by Suvraneel** [comment]
@ank1traj  **That SOLID, DRY principle thing ka single meaning/significance hi samjhao na tum phir... Multiple nai chaiye.**

Remember We're all volunteers here... not employees. Unnecessary remarks are not appreciated.
Opensource necessitates finding other's mistakes with empathy

Anyway, have a good day. 
CC: @its-puneet

**Comment by ank1traj** [comment]
@Suvraneel That DRY principal ka matlab ye tha ki sometimes code following everything can be missed and cause some issues. So, check the deployment before merging & after merging the code. And, I didn't add only **DRY**. I added SOLID and etc. also by not considering you. But, if you wanna take it on you only then I am sorry, I can't help you in this.


And, yeah one thing What was added there it was for everyone and not for you only. So, stop taking it on yourself. Rest is your wish I can't explain it anymore. I already said you sorry if you taken it on yourself. But, now I can't say anything further. 


Take care. Have a good day. 
Thanks.

Cc: @its-puneet
